docs/LARGE_TESTS.md

```diff
@@ -0,0 +1,165 @@
+# Large Tests: Building and Testing
+
+This guide gets you started with writing and running large tests on Snap plugins. If your plugin needs updated to the large test framework you can find step by step instructions [here](https://github.com/kjlyon/snap/blob/master/Pluginsync_for_large_tests.md).
+
+## Effective Large Tests
+
+As of 1.0 we introduced a large test framework which can be added to your plugin by running our pluginsync tool. The default large test performs the following actions:
+* Use environment variable to populate docker compose specification in: `scripts/test/docker_compose.yml`
+* Download the latest containers via `docker pull` and run them
+* Conditionally run `scripts/test/setup.rb` before any test (use this to create test database, test service etc)
+* Download and run the appropriate version of Snap per `$SNAP_VERSION`
+* Scan `examples/task/*.yml` for list of tasks and metrics
+* Load Snap plugins, first from the local `build/linux/x86_64/*` directory, then try `build.snap-telemetry.io` s3 bucket
```

Comment:
Should we include the link to the s3 bucket here (http://snap.ci.snap-telemetry.io)